docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md

```diff
@@ -1,6 +1,6 @@
 # Delete Edge Syntax
 
-Given an edge type, the source vertex and the dest vertex, Nebula supports `DELETE` the edge, its associated properties, the edge rank, and edge with a certain rank, the syntax as the follows:
+The `DELETE EDGE` statement is used to delete edges. Given an edge type, the source vertex and the dest vertex, Nebula supports `DELETE` the edge, its associated properties and the edge ranking. You can also delete an edge with a certain rank, the syntax as the follows:
 
 ```ngql
 DELETE EDGE <edge_type> <vid> -> <vid> [, <vid> -> <vid> ...]
```

Comment:
I think there is a ranking there.
I'm not sure it's merged in another pr

Reply:
Yes, it's in another pr